reference/mongodb/mongodb/driver/serverapi.xml

```diff
@@ -0,0 +1,179 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- $Revision$ -->
```

Comment:
Is this still relevant for doc files now that SVN is no longer being used?

Reply:
Other files retained it, so I was conservative in removing it (read: I didn't bother after copy/pasting files). I'll check in the SO chat to see if this can be dropped.